backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -364,6 +393,104 @@ export class TransactionLinkResolver {
     }
   }
 
+  @Authorized([RIGHTS.QUERY_REDEEM_JWT])
+  @Mutation(() => String)
+  async createRedeemJwt(
+    @Arg('gradidoId') gradidoId: string,
+    @Arg('senderCommunityUuid') senderCommunityUuid: string,
+    @Arg('senderCommunityName') senderCommunityName: string,
+    @Arg('recipientCommunityUuid') recipientCommunityUuid: string,
+    @Arg('code') code: string,
+    @Arg('amount') amount: string,
+    @Arg('memo') memo: string,
+    @Arg('firstName', { nullable: true }) firstName?: string,
+    @Arg('alias', { nullable: true }) alias?: string,
+    @Arg('validUntil', { nullable: true }) validUntil?: string,
```

Comment:
Thanks for the work on this! One thing I'd suggest is to consider consolidating the many function parameters into a single DTO or input object. This will improve readability, reduce the risk of mistakes when passing so many arguments around, and make future maintenance easier.

Having so many individual parameters can get quite verbose and error-prone, especially as the codebase grows. Using an input type would also align better with GraphQL best practices and keep the resolver signature cleaner.